packages/effects-docs/gatsby-node.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/node-apis/
+ */
+const { createFilePath } = require(`gatsby-source-filesystem`);
+const { toExamples, withoutExampleSuffix } = require("./src/util/examples.js");
+const fs = require("fs");
+const path = require("path");
+const examplesGql = `
+{
+  examples: allFile(filter: {
+    relativePath: {
+      regex: "/examples/.*.example$/"
```

Comment:
even if something _isnt required_--gatsby tries to parse it.  v. obnoxious--forced me to write. `x.js` files as `x.js.example`

Reply:
...due to effects.  i _guess_ i could wire gatsby's babel to _our_ babel 🤔 . another time.